src/templates/template-vision-segmentation/trainers.py

```diff
@@ -3,7 +3,14 @@
 import ignite.distributed as idist
 import torch
 from data import prepare_image_mask
-from ignite.engine import DeterministicEngine, Engine, Events
+
+#::: if(it.deterministic) { :::#
+from ignite.engine import DeterministicEngine, Engine, Events  # usort: skip
+
+#::: } else { :::#
+from ignite.engine import Engine, Events
+
+#::: } :::#
```

Comment:
@theory-in-progress please add return type hint to `setup_trainer` and `setup_evaluator`

Reply:
Should the changes to `setup_trainer` and `setup_evaluator` be in the same PR?

Reply:
Just return type hints